### PR TITLE
runtest.py: Fix traceback print

### PR DESCRIPTION
**Pull request by yamt**

### CODE REVIEW

File: tests/wamr-test-suites/spec-test-script/all.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ def test_case(
         CMD,
         bufsize=1,
         stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE,
+        stderr=subprocess.DEVNULL,
```
Comment (lum1n0us) [inline tests/wamr-test-suites/spec-test-script/all.py:158]: if so, we won't receive error information, and it may limit our choice.

Comment (yamt) [reply tests/wamr-test-suites/spec-test-script/all.py:158]: do you mean it's better to keep the unused stderr plumbing?

File: tests/wamr-test-suites/spec-test-script/runtest.py [hunk 2]
```diff
@@ -1274,7 +1274,7 @@ def test_assert_with_exception(form, wast_tempfile, wasm_tempfile, aot_tempfile,
             else:
                 raise Exception("unrecognized form '%s...'" % form[0:40])
     except Exception as e:
-        traceback.print_exc()
+        traceback.print_exc(file=sys.stdout)
```
Comment (loganek) [inline tests/wamr-test-suites/spec-test-script/runtest.py:1277]: hi @yamt , could you clarify what is the benefit of this change? Whereas I understand what it does, it's not quite clear why do we want to change it?

**Comment by wenyongh** [comment]
Close this PR since it hasn't been updated for a long time, and the changes seem unneeded.

**Pull request closed without merging**